crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -220,6 +220,8 @@ pub(crate) async fn farm_multi_disk(
         }).fuse() => {},
     );
 
+    let _ = tokio::task::spawn_blocking(|| drop(single_disk_plots)).await;
```

Comment:
There are more implicit blocking operations in this function, the whole thing needs to run in its own thread if it is important for the app.

Reply:
Joining in drop is blocking so it might block the async runtime.